src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -96,32 +119,64 @@ export const Browser: FunctionComponent<BrowserProps> = (
   };
 
   useEffect(() => {
-    const { stableId } = props.match.params;
-    const location = props.location.search;
+    if (props.match.params.genomeId === undefined) {
+      const locationStr = getChrLocationStr(
+        props.chrLocation[props.activeGenomeId]
+      );
+
+      urlFor.browser(
+        props.activeGenomeId,
+        props.activeEnsObjectId[props.activeGenomeId],
+        locationStr
+      );
+    } else {
+      setTrackStatesFromStorage(browserStorageService.getTrackStates());
+    }
+  }, []);
```

Comment:
I guess this is temporary solution until https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-122

Reply:
Yes, it is only temporary.

Reply:
@azangru I have created the intermediate view but how it is related to the ticket ENSWBSITES-122?

Reply:
Now that I'm re-reading this piece of code, I am not sure what I meant. I probably was confused. But the if-block of this code isn't doing anything at the moment? It just generates a url string and returns `undefined`?